### DISCUSSION TITLE
Docs about installation

### DISCUSSION THREAD

**Discussion by krokusmoen** [lucky-media/landtamic#3] [General]
Hello,

Exciting starter kit. Having trouble installing it, gets this message.

Validating starter kit license...
License required for [lucky-media/landtamic]!
This is a paid starter kit. If you haven't already, you may purchase a license at:
https://statamic.com/starter-kits/luckymedia/landtamic

We have bought a license, but could not find any documentation on what to put in our .env file (I think that is what is missing?)

**Comment by arlindmusliu** [comment]
Hey @krokusmoen ,

You should paste the licence key and it should install the starter kit.

Here you have more info on installing Statamic Starter Kits:
[https://statamic.dev/starter-kits/installing-a-starter-kit](https://statamic.dev/starter-kits/installing-a-starter-kit)

Let us know if you managed to get it done.

Thanks

**Reply by krokusmoen** [reply]
I installed a fresh Statamic through composer, then followed the instructions with this command:

`php please starter-kit:install lucky-media/landtamic
`

It does not prompt for license. Is there another way to do it?

**Reply by lokmanm** [reply]
Hi @krokusmoen,

You should be prompted to enter a license key after doing `php please starter-kit:install lucky-media/landtamic`.

If for some reason this fails I believe this is out of our hands and you should reach out to Statamic help.

Starter kit installing / license validation is on their end.

Let me know if anything else comes up, 

Thanks

**Reply by krokusmoen** [reply]
Hi @lokmanm ,

Got it working through composer install. 

The current Statamic needs to be Pro and a license should be present in the .env file.

**Reply by lokmanm** [reply]
Hey @krokusmoen, I am glad you got it working. 

A bit confused why Statamic needs to be Pro as we have configured it in a way that doesn't require Pro (unless you need it for the other features).
